internal/util/projutil/project_util.go

```diff
@@ -113,9 +113,19 @@ func getHomeDir() (string, error) {
 	return homedir.Expand(hd)
 }
 
+// TODO(hasbro17): If this function is called in the subdir of
```

Comment:
Should be here the best place for it? WDYT about track an issue as RFE for that with all info as to how to reproduce the scenario and the goal? Then, we can try to analyse it and see if it is possible or not?

Reply:
@camilamacedo86 I'm planning to compile the entire list of TODOs here into a follow up ticket/issue for more general refactoring work. 
If I stop and file an issue for every TODO I come across this PR will never end, so I'm trying to keep the scope minimal for now.

That being said, this particular caveat is not an issue right now. So there's no "issue" to reproduce.
It's something that could be an issue if this util is used with the wrong expectation. Which is why I've only noted this down with the expectation of addressing this later if that's even possible.